### PR TITLE
Update keras.md as suggested by Pull Request #301

### PR DESCRIPTION
**Pull request by sryu1**
As suggested by https://github.com/googlecreativelab/teachablemachine-community/pull/301, Defined class_names, added end='' for print class (line 43) since using readlines() for labels.txt adds an extra line between class and confidence score. Added compile=False since the model is already trained. Fixed a couple of syntax (e.g. #301 is missing some things in line 7), also removed scientific notation for clarity.

### CODE REVIEW

**Comment by sryu1** [comment]
@HalfdanJ Could you pls merge this?

**Comment by HalfdanJ** [comment]
Thanks a lot @sryu1 !
